fastspell/fastspell.py

```diff
@@ -248,7 +275,7 @@ def perform_identification(args):
     fs = FastSpell(args.lang, mode=mode, hbs=args.hbs, script=args.script)
     
     for line in args.input:        
-        lident = fs.getlang(line)
+        lident = fs.getlang(line.lower())
```

Comment:
Strongly disaggree with this lowercasing, since casing is used in `remove_unwanted_words`.